src/libxrpl/protocol/LedgerFormats.cpp

```diff
@@ -364,6 +364,36 @@ LedgerFormats::LedgerFormats()
             {sfPreviousTxnLgrSeq,   soeREQUIRED}
         },
         commonFields);
+		
+	add(jss::MPTokenIssuance,
+        ltMPTOKEN_ISSUANCE,
+        {
+            {sfIssuer,               soeREQUIRED},
+            {sfSequence,             soeREQUIRED},
+            {sfTransferFee,          soeDEFAULT},
+            {sfOwnerNode,            soeREQUIRED},
+            {sfAssetScale,           soeDEFAULT},
+            {sfMaximumAmount,        soeOPTIONAL},
```

Comment:
`MaximumAmount` is marked required in the spec.

Reply:
IMHO, the spec should be updated for `MaximumAmount`. We don't know how popular "limitless" MPTs will be, but there's no need to store 64-bits of 1, when "not present" can just as easily mean "no limit", and thus, the limit will be 64-bit maxint. It's treated that way in `rippleSendMPT`.

Reply:
Yes this is a mistake in the spec, MaximumAmount should be a field that defaults to the largest unsigned 63 bit integer if not specifed